custom_button/CustomButton.py

```diff
@@ -0,0 +1,101 @@
+import tkinter as tk
+from tkinter import *
+from external.constants import FONT_FAMILY, FONT_SIZE
```

Comment:
Okay so this error occurs, because you have `main.py` file in `custom_button`.
Please move this file into our main directory (`TkinterProject`), in that case path `external.constants` will be okay!